webui-src/app/people/people_util.js

```diff
@@ -1,13 +1,43 @@
-const m = require('mithril');
 const rs = require('rswebui');
-
+const m = require('mithril');
 function checksudo(id) {
   return id === '0000000000000000';
 }
 
+const UserAvatar = () => ({
+  view: (v) => {
+    const imageURI = v.attrs.avatar;
+    console.log(imageURI);
+    return imageURI === undefined || imageURI.mData.base64 === ''
+      ? []
+      : m('img.avatar', {
+          src: 'data:image/png;base64,' + imageURI.mData.base64,
+        });
+  },
+});
+
+function contactlist(list) {
+  const result = [];
+  if (list !== undefined) {
+    list.map((id) => {
+      id.isSearched = true;
+      rs.rsJsonApiRequest('/rsIdentity/isARegularContact', { id: id.mGroupId }, (data) => {
+        if (data.retval) result.push(id);
+        console.log(data);
+      });
+    });
+  }
+  return result;
+}
+
 function sortUsers(list) {
   if (list !== undefined) {
-    const result = [...list];
+    const result = [];
+    list.map((id) => {
+      id.isSearched = true;
+      result.push(id);
+    });
```

Comment:
Tiny nit for readability:
```suggestion
    const result = list.map((id) => {
      id.isSearched = true;
      return id;
    });
```
Or even simpler:
```suggestion
    const result = list.map((id) => ({
      ...id,
      isSearched: true,
    }));
```

Reply:
@rottencandy  search operation is not when i tried to add the above change.

Reply:
Hmm, Probably due to the inplace updates. Pretty sure atleast the first method would work, but this is not a big deal.